protocol/src/main/java/net/md_5/bungee/protocol/packet/ResourcePackResponse.java

```diff
@@ -0,0 +1,47 @@
+package net.md_5.bungee.protocol.packet;
+
+import io.netty.buffer.ByteBuf;
+import lombok.AllArgsConstructor;
+import lombok.Data;
+import lombok.EqualsAndHashCode;
+import lombok.NoArgsConstructor;
+import net.md_5.bungee.protocol.AbstractPacketHandler;
+import net.md_5.bungee.protocol.DefinedPacket;
+import net.md_5.bungee.protocol.ProtocolConstants;
+
+@Data
+@NoArgsConstructor
+@AllArgsConstructor
+@EqualsAndHashCode(callSuper = false)
+public class ResourcePackResponse extends DefinedPacket
+{
+
+    private String hash;
+    private int result;
+
+    @Override
+    public void read(ByteBuf buf, ProtocolConstants.Direction direction, int protocolVersion)
+    {
+        if ( protocolVersion < ProtocolConstants.MINECRAFT_1_10 )
+        {
+            hash = readString( buf );
```

Comment:
How does the absence of this affect functionality of the proxy that relies on the hash?

Reply:
It doesn't. Nothing is exposed to use the response hash for anything as that is the same as the one sent in the request and that is used instead.